float64[] position # xyzYPR: position + eluer angle
float64[] force # human force
